### PR TITLE
Update bridge documentation to include 'enabledad'

### PR DESCRIPTION
**Pull request by MikeZappa87**
Updating the bridge documentation to include 'enabledad'

Related Issue: 
https://github.com/containernetworking/plugins/issues/680

Related PR:
https://github.com/containernetworking/plugins/pull/695

### CODE REVIEW

File: content/plugins/current/main/bridge.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ If the bridge is missing, the plugin will create one on first use and, if gatewa
 * `ipam` (dictionary, required): IPAM configuration to be used for this network. For L2-only network, create empty dictionary.
 * `promiscMode` (boolean, optional): set promiscuous mode on the bridge. Defaults to false.
 * `vlan` (int, optional): assign VLAN tag. Defaults to none.
+* `enabledad` (boolean, optional): enables duplicate address detection for the container side veth. Defaults to false.
```
Comment (AkihiroSuda) [inline content/plugins/current/main/bridge.md:64]: Perhaps it would be better to describe the CNI plugin version with this feature? (v1.0.2? v1.1.0?)

Comment (MikeZappa87) [reply content/plugins/current/main/bridge.md:64]: It's def a solid idea. Its funny Mike Brown and I were just discussing versioning around all of this ha! Are you suggesting adding the version to the 'enabledad' description?

Comment (AkihiroSuda) [reply content/plugins/current/main/bridge.md:64]: > Are you suggesting adding the version to the 'enabledad' description?

Yes 👍

Comment (MikeZappa87) [reply content/plugins/current/main/bridge.md:64]: Right on. I just saw your description for the firewall plugin, that def alleviates the confusion :+1:

**Pull request merged**